src/App.tsx

```diff
@@ -15,49 +18,86 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
 export const App: React.FC = () => {
+  const [isReversed, setDirection] = useState(false);
```

Comment:
Let's follow the naming convention for the useState. 
`const [value, setValue] = useState()`

```suggestion
  const [isReversed, setIsReversed] = useState(false);
```